pyproject.toml

```diff
@@ -4,6 +4,9 @@ target_version = ['py39', 'py310', 'py311']
 skip-string-normalization = true
 skip-magic-trailing-comma = true
 
+[tool.pytest]
+markers = ["rigetti_integration: tests that connect to Quil compiler or QVM.", "slow: mark tests as slow", "weekly: mark tests as run only by weekly automation"]
```

Comment:
This should be in the `[tool.pytest.ini_options]` section below.
Use `pytest --markers` to check if the extra marks are available.

Also the list should be formatted as one item per line for a better readability.